dojo/unittests/scans/twistlock/scan_report_many_vuln.csv

```diff
@@ -0,0 +1,5 @@
+Registry,Repository,Tag,Id,Scan Time,Pass,Type,Distro,Hostname,Layer,CVE ID,Compliance ID,Type,Severity,Packages,Source Package,Package Version,Package License,CVSS,Fix Status,Description,Cause,Published,Custom Labels
+provisioner-docker.cloud.thdm.local,acme/acme-image-root,1.1.2,sha256:09951c12314543758569898475787989070600790978987567dcd68248372aac,14:24.3,TRUE,cli,FancyOS,docker-dev-1234567a-1234-1234-abcd-12345567890a,,CVE-2023-12334,98,java,critical,io.class_class,,1.1.1.Release,,1.2,fixed in 1.2.11,"HttpObjectDecoder.java in a complex class before 1.2.11  allows an intruder to do horrible things that will end in  \""Bad thing.\""",,12:00.0,
+provisioner-docker.cloud.thdm.local,acme/acme-image-ultimate,1.1.2,sha256:09951r3412f122121dcbº341212341bc111231233123121231dcd68248372aac,10:11.2,TRUE,somecli,GoodOS,docker-image-som-987777b-4312-12ty-2133-42343423542f,,CVE-2035-1000,0º,langua,high,cool framework_cool-core,,1.1.1,,1.1,"fixed in the future 1.0., 2.3.45","Cool Framework, versions 0.5.0 prior to 0.1.0.  and older unsupported versions, provide client-side support for multi requests. When  fractals are represented, the hyper speed fails",,29:00.0,
+provisioner-docker.,acme/acme-image-beta,1.1.2,sha256:c7012312354352356542634455425435524523452352345345555520a899c91e,98:76.5,TRUE,cli,SantaCruzOS,build-123-123-123,,CVE-2030-12345,21,old-language.high,load,,1.13.4,,5.6,fixed in >=0.0.1,"A prototype tjat will expose a black hole was found ",,49:00.0,

```

Comment:
typo `old-language.high` should read `old-language,high`. My test manual import fails on it.